### PR TITLE
Fix/responsive mobile

### PR DESCRIPTION
**Pull request by duchuytb9x**
## Description

Change some Pages to meet Responsive Mobile

## Test cases:

Please describe tested cases.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvements (ie: code cleaning or remove unused codes or performance issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI Changes

Add before and after changes for UI if needed.

|before|after|
|-|-|
|||
|||
|||
|||
|||
|||
|||
|||

### CODE REVIEW

**Comment by HuberTRoy** [comment]
Thanks for your contribute, a little more need to change: 

1. The align of project detail looks not same as before, can you please fix this?

image

image

2. substr is a deprecated method, can you please change a method to do?

File: src/pages/bridge/index.tsx [hunk 1]
```diff
@@ -537,7 +537,7 @@ const BridgeInner: FC = () => {
         {currentTab === 'baseToEth' && (
           <>
             <div className={styles.bridgeContent}>
-              <div className="col-flex" style={{ alignItems: 'center', gap: 8, minWidth: 500 }}>
+              <div className="col-flex" style={{ alignItems: 'center', gap: 8, minWidth: '-webkit-fill-available' }}>
```
Comment (HuberTRoy) [inline src/pages/bridge/index.tsx:540]: I think use `width: 100%` is better.

Comment (duchuytb9x) [reply src/pages/bridge/index.tsx:540]: ok, I will look again

**Comment by duchuytb9x** [comment]
1. I will edit the project details so that the layout does not change
2. Can I change the DeploymentId to aaa...bbb?
Because on mobile the text is too long

**Comment by HuberTRoy** [comment]
@duchuytb9x 
Yes, shrink the deploymentId is good. Just don't use `substr`, use `slice` or something would better than a deprecated method. Also if can add more type asset would more better: e.g: ```if (!isString(value)) return value```

**Comment by duchuytb9x** [comment]
There is a table in the project's Indexer List that requires additional scrolling. I see you are ToDo something

**Comment by duchuytb9x** [comment]
I have just changed the code according to the review

**Comment by duchuytb9x** [comment]
@HuberTRoy Almost every necessary page has been changed to be responsive for mobile

**Comment by HuberTRoy** [comment]
Made some tiny changes. Very thanks for the great work~.

**Pull request merged**